src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -291,7 +293,7 @@ static ucs_status_t uct_mlx5_iface_init_dm_tl(uct_mlx5_dm_data_t *data,
     struct ibv_exp_reg_mr_in mr_in;
 
     data->seg_len   = ucs_align_up(config->dm.seg_len, 8);
-    data->seg_count = ucs_align_up(config->dm.count, 8);
+    data->seg_count = config->dm.count;
```

Comment:
maybe use sizeof(uint64_t) instead of 8?
need to somehow connect this parameter to the memory copy function which uses uin64_t-based copy
for example if we would use SSE/AVX in the memory copy, then would need to align to 16/32/64B

Reply:
right, I will update this value by sizeof of header (max AM or TM). for now there is no such structures because send routines are out of this PR

Reply:
and SSE/AVX could not be used because source/destination data are not aligned